### PR TITLE
added test code

### PR DESCRIPTION
**Pull request by will-wow**
Just a test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Tracking FEC Contribution Data
 pip -r requirements.txt
 ```
 
+Just a test
```
Comment (cgarretty) [inline README.md]: Why did you add this?

**Comment by will-wow** [comment]
This was just a test.
